ui/src/shared/actions/notifications.ts

```diff
@@ -1,20 +1,43 @@
-import * as NotificationsActions from 'src/types/actions/notifications'
-import * as NotificationsModels from 'src/types/notifications'
+import {Notification} from 'src/types'
 
-export const notify: NotificationsActions.PublishNotificationActionCreator = (
-  notification: NotificationsModels.Notification
-): NotificationsActions.PublishNotificationAction => ({
-  type: 'PUBLISH_NOTIFICATION',
+export type Action =
+  | PublishNotificationAction
+  | DismissNotificationAction
+  | DismissAllNotificationsAction
+
+export enum ActionTypes {
```

Comment:
Is there any reason to make a separate `ActionTypes` enum rather than embedding a string literal type in the `Action` types directly? My take is that it provides no benefit and is superfluous boilerplate.... but I haven't been able to get consensus on this yet. nbd either way.

https://influxdata.slack.com/archives/C9RL1S494/p1542669173011600

Reply:
oh hmm. that does make sense.